Examples/CodePushDemoApp/CodePushDemoAppTests/InstallUpdateTests/testcases/IsPendingTest.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
```

Comment:
Should we also add a test for checking `isPending` after a call to `getCurrentPackage` since that is actually populated differently and is the most common case?